src/snowflake/snowpark/_internal/utils.py

```diff
@@ -803,6 +811,22 @@ def get_copy_into_table_options(
     return file_format_type_options, copy_options
 
 
+def get_copy_into_location_options(
```

Comment:
nit: the signature name implies it's only returning copy_into_location_options but this function also returns file_format_type_options, or file_format_type_options is part of copy_into_location_options?

Reply:
this is true. I just followed the same pattern for copy into table. I can refactor both functions so they match